### PR TITLE
Dev magda issue 220

### PR DESCRIPTION
**Pull request by eynimeni**
this is still work in progress...

### CODE REVIEW

File: classes/catscale.php [hunk 1]
```diff
@@ -416,13 +418,14 @@ public function get_testitems(
      * @param int $contextid
      * @param mixed $question
      * @param bool $includesubscales
+     * @param mixed $catscaleid
```
Comment (bernhard-wunderbyte) [inline classes/catscale.php:421]: id sollte ein int sein

**Comment by cbadusch** [comment]
adding a comment :)

**Comment by WunderJacob** [comment]
Awesome work

**Comment by dasistwas** [comment]
great work in progress

**Comment by mona-shakiba** [comment]
a Code review

**Review by bernhard-wunderbyte — COMMENTED** [review]
Done

**Pull request closed without merging**